ActionMenu/ActionMenu.csproj

```diff
@@ -21,5 +21,15 @@
         <EmbeddedResource Include="UIResources\icon_avatar_emotes.svg" />
         <EmbeddedResource Include="UIResources\icon-melon.svg" />
         <EmbeddedResource Include="UIResources\icon_avatar_settings_profile.svg" />
+    
+        <EmbeddedResource Include="UIResources\icon_media_fwd.svg" />
+        <EmbeddedResource Include="UIResources\icon_media_pause.svg" />
+        <EmbeddedResource Include="UIResources\icon_media_play.svg" />
+        <EmbeddedResource Include="UIResources\icon_media_prev.svg" />
     </ItemGroup>
+
+    <Target Name="Deploy" AfterTargets="Build">
+        <Copy SourceFiles="$(TargetPath)" DestinationFolder="C:\Program Files (x86)\Steam\steamapps\common\ChilloutVR\Mods\" />
+        <Message Text="Copied $(TargetPath) to C:\Program Files (x86)\Steam\steamapps\common\ChilloutVR\Mods\" Importance="high" />
+    </Target>
```

Comment:
If properly setup, the project will copy the dll directly into cvr mods directory. This indicates your setup is incorrect?
Again something I cannot merge.